config/client.json

```diff
@@ -26,5 +26,6 @@
   "siftscience_key",
   "facebook_api_key",
   "discover_blog_id",
-  "support-user"
+  "support-user",
+  "persist-redux"
```

Comment:
This is not necessary, because `persist-redux` is a feature flag. Features are already exposed to the client under the `features` config key.